quantum/pointing_device/pointing_device_auto_mouse.c

```diff
@@ -31,63 +31,75 @@ static bool is_mouse_record(uint16_t keycode, keyrecord_t* record) {
     return false;
 }
 
+/* handle mouskey event */
+void auto_mouse_keyevent(bool pressed) {
+    pressed ? local_auto_mouse.status.mouse_key_tracker++ : local_auto_mouse.status.mouse_key_tracker--;
+    local_auto_mouse.timer.active = timer_read();
+}
+
+void auto_mouse_reset_trigger(void) {
+    local_auto_mouse.status.mouse_key_tracker = 0;
+    local_auto_mouse.timer.active             = 0;
+    local_auto_mouse.timer.delay              = timer_read;
```

Comment:
Shouldn't this be:

local_auto_mouse.timer.delay              = timer_read();

Reply:
no the delay timer is the delay for when a non mouse key is hit it will delay the auto mouse layer activating.
the active timer is what needs to be reset as once that expires the layer will be deactivated